about.md

```diff
@@ -0,0 +1,217 @@
+
+```
+---
+Title: YAML Example
+Description: This is what your Metadata will look like in YAML
+Author: Your Name
+Date: 2016/05/10
+---
+```
+
+## Flexibility and Customization
+
+Pico is not just easy to use, it's *powerful*!  Pico has two very large avenues of customization, [Themes and Plugins][Customization].
+
+### Theming With Twig Templates
+
+Pico's default theme is *not* intended for production use.  It is provided as a great, but minimal starting place for you to develop your own customized website.  If you aren't familiar with HTML, fear not, we have an ever-growing variety of community-created themes available [here on our site][SiteThemes], as well as some more on [our wiki][WikiThemes].
+
+Pico's "Themes" are built using [Twig Templates][Twig].  Twig is a template engine which provides an easy and powerful way to customize your website.  You can use small amounts of Twig to add dynamic content to a mostly-static HTML website, or use large amounts of Twig to build in some really incredible features.  Twig is so powerful that you'll find it can accomplish most tasks by itself, eliminating the need for plugins.  You can find more information on making your own templates in [Twig's Documentation][TwigDocs] and see how they relate to Pico in our own [Documentation][DocsThemes].
+
+{% include gallery.html gallery='themes' %}
+
+And if that's still not enough creative power for you, you can check out Pico's Plugin system.
+
+### Plugins Extend Pico's PHP
+
+Pico's Plugin system allows for users to extend Pico's functionality by hooking in their own PHP code.  Along with Themes, we also have a growing library of community-developed plugins you can use to add new features to your Pico site.  You can find these plugins [here on our site]({{ site.github.url }}/customization/#plugins), and even more on [our wiki]({{ site.gh_project_url }}/wiki/Pico-Plugins).
+
+## Getting Started
+
+Ready to try Pico for yourself?  Head on over to our [Download][Download] page to get yourself a copy.  You can find more information on building your own Pico site in our [Documentation][Docs].  And of course, **we're here to help**!  If you require any assistance, or if you find a bug in Pico, let us know!  Check out the [Getting Help][GettingHelp] section of our Documentation for more details.  We appreciate your feedback!  Pico is a community-driven project and we need *your* feedback to keep making it better!
+
+{% comment %}
+---
+
+* **General Notes**
+* Include a larger "About" portion at the beginning?
+  * Should contain all the "Why's" of Pico, before the "How's".
+  * Explain what Pico is better *before* talking about other CMS's.
+* Better image captions (Already updated some of them.)
+
+* **Disadvantages of Other CMS's**
```

Comment:
Wasn't it your girlfriend who said that WordPress's configuration overwhelmed her? This (not your girlfriend in particular, but that "too complicated, don't need it" :smile:) should be covered somewhere.